pkg/console/operator/operator.go

```diff
@@ -138,15 +134,23 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 	logrus.Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
 	defer logrus.Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
 
-	operatorConfig := obj.(*consolev1.Console)
-	if err := c.handleSync(operatorConfig); err != nil {
+	operatorConfig := obj.(*operatorsv1.Console)
+
+	consoleConfig, err := c.consoleConfigClient.Get(api.ResourceName, metav1.GetOptions{})
+	if errors.IsNotFound(err) && CreateDefaultConsoleFlag {
+		c.consoleConfigClient.Create(c.defaultConsoleConfig())
```

Comment:
Shouldn't we handle errors here?

Reply:
I kept the pattern of create, return the error, let it die, and kick the loop again to check next time.

Reply:
Right, but we're not logging any error if the create itself fails. We should at least log that so we know for debugging.